data/language/dutch.txt

```diff
@@ -584,7 +584,7 @@ STR_0582    :
 STR_0583    :
 STR_0584    :
 STR_0585    :
-STR_0586    :Bootvormige karretjes lopen over achtbaanrails, waardoor ze hellende bochten en diepe afdalingen kunnen maken, waarna ze in bakken water neerplonzen en rustig een stukje kunnen varen
+STR_0586    :Bootvormige karretjes rijden over een achtbaanrails, waardoor ze hellende bochten en diepe afdalingen kunnen maken, waarna ze in bakken water neerplonzen en rustig een stukje kunnen varen
```

Comment:
Ik heb het niet over rails maar over achtbaanrails, jullie zien toch wel het verschil tussen rails=spoorstaven en rails=spoor :s. Wat jullie eigenlijk hier zeggen is dat een bootvormig karretje tegelijkertijd over meerdere sporen rijdt?

Maar hoe dan ook ik heb 'een' maar weggehaald want dat vinden we tenslotte allemaal goed.